tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>either it is an attribute, so <code>other_simsat.config</code></p><p> or it is a method then needs to have a verb in the name:</p><p><code>other_simsat.get_config()</code></p><p> <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>

Reply:
<p>The methods are now <code>get_config()</code>  and <code>get_class_config()</code>  <span class="ql-cursor">﻿</span></p>